### PR TITLE
.NET Sentry SDK v3.0.0 - ASP.NET integration

### PR DESCRIPTION
**Pull request by bruno-garcia**
To be merged once .NET 3.0 comes out

### CODE REVIEW

File: src/platforms/dotnet/guides/aspnet/index.mdx [hunk 1]
```diff
@@ -3,18 +3,18 @@ title: ASP.NET
...
-Sentry provides an integration with _ASP.NET_ through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry).
+Sentry provides an integration with _ASP.NET_ through the [Sentry.AspNet NuGet package](https://www.nuget.org/packages/Sentry.AspNet).
 
 ## Install
 
 Add the Sentry dependency:
 
 ```powershell {tabTitle:Package Manager}
-Install-Package Sentry -Version {{ packages.version('sentry.dotnet') }}
+Install-Package Sentry.AspNet -Version {{ packages.version('sentry.dotnet') }}
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aspnet/index.mdx:13]:
```suggestion
Install-Package Sentry.AspNet -Version {{ packages.version('sentry.dotnet.aspnet') }}
```

File: src/platforms/dotnet/guides/aspnet/index.mdx [hunk 2]
```diff
@@ -3,18 +3,18 @@ title: ASP.NET
...
 ```
 
 ```shell {tabTitle:.NET Core CLI}
-dotnet add package Sentry -v {{ packages.version('sentry.dotnet') }}
+dotnet add package Sentry.AspNet -v {{ packages.version('sentry.dotnet') }}
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aspnet/index.mdx:17]:
```suggestion
dotnet add package Sentry.AspNet -v {{ packages.version('sentry.dotnet.aspnet') }}
```

**Review by marandaneto — APPROVED** [review]
LGTM

**Review by PeloWriter — APPROVED** [review]

File: src/platforms/dotnet/guides/aspnet/index.mdx [hunk 3]
```diff
@@ -39,6 +39,7 @@ public class MvcApplication : System.Web.HttpApplication
         // Add this to Application_Start
         _sentry = SentrySdk.Init(o =>
         {
+            o.AddAspNet();
             o.Dsn = new Dsn("___PUBLIC_DSN___");
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aspnet/index.mdx:43]: This also changed.
`Dsn` class is internal and a String is passed only. Needs fixing.

**Pull request merged**